File: python/Nim_Array.py
nim = "2510651"  

# Membuat array digit
digits = []
for d in nim:
    digits.append(int(d))

# Perhitungan
total = sum(digits)
maksimum = max(digits)
minimum = min(digits)
rata_rata = total / len(digits)
reverse_array = digits[::-1]

# Output
print("Array digit        :", digits)
print("Total digit        :", total)
print("Digit maksimum     :", maksimum)
print("Digit minimum      :", minimum)
print("Rata-rata digit    :", rata_rata)
print("Reverse array NIM  :", reverse_array)

File: php/Nim_Array.php
<?php

echo "Masukkan NIM: ";
$nim = trim(fgets(STDIN));

$digits = [];
for ($i = 0; $i < strlen($nim); $i++) {
    $digits[] = intval($nim[$i]);
}

$total = array_sum($digits);
$max = max($digits);
$min = min($digits);
$rata = $total / count($digits);
$reverse = array_reverse($digits);

echo "Array digit        : [" . implode(", ", $digits) . "]\n";
echo "Total digit        : $total\n";
echo "Digit maksimum     : $max\n";
echo "Digit minimum      : $min\n";
echo "Rata-rata digit    : $rata\n";
echo "Reverse array NIM  : [" . implode(", ", $reverse) . "]\n";
?>